edgelet/edgelet-docker/src/runtime.rs

```diff
@@ -1274,13 +1286,65 @@ fn get_notary_parameters(
     Some((notary_auth, gun, tag, config_path.to_path_buf()))
 }
 
+// Disallow adding privileged and other capabilities if allow_privileged_docker_containers is false
+fn unset_privileged(
+    allow_privileged_docker_containers: bool,
+    create_options: &mut ContainerCreateBody,
+) {
+    if let Some(config) = create_options.host_config() {
+        if !allow_privileged_docker_containers
+            && (config.privileged() == Some(&true) || !config.cap_add().is_empty())
```

Comment:
can cap add container a different capability than "privileged"?
In which case, you might still want to drop privileged flag even if cap add is not empty.

Reply:
No, its an OR. If we don't allow privileged, and either the privileged flag is set or any capability is added, clear the privileged flag and reset capabilities.